test/integration.test.ts

```diff
@@ -600,18 +601,62 @@ const testCases: {
       expect(tableHeads).toContain('Exports')
       expect(tableHeads).toContain('File')
       expect(tableHeads).toContain('Size')
+      const [exportsIndex, fileIndex, sizeIndex] = [
+        tableHeads.indexOf(pc.underline('Exports')),
+        tableHeads.indexOf(pc.underline('File')) - 9, // pc.underline('File').length = 4  + 'File'.length + 5
+        tableHeads.indexOf(pc.underline('Size')) - 18,
+      ]
 
+      // meet trouble to get index
```

Comment:
Is this line by purpose related?
```suggestion
      // meet trouble to get index
```